server/src/main/java/org/spine3/server/entity/idfunc/Producers.java

```diff
@@ -81,9 +81,11 @@ private Producers() {
      *
      * @param <I> the type of entity IDs
      */
-    private static class ProducerFromFirstEventMessageField<I> implements IdSetEventFunction<I, Message> {
+    private static class ProducerFromFirstEventMessageField<I>
+                   implements IdSetEventFunction<I, Message> {
```

Comment:
Do not make indents manually. Use `Idea`s `reformat code` to fix the indents like this. Check the similar cases.